### DISCUSSION TITLE
my app has the same problem on ubuntu 2204

### DISCUSSION THREAD

**Discussion by Anderson-Vilela-op** [prisma/prisma#21059] [Q&A]
This is not a specific error of ubuntu, I am using wsl2 with ubuntu and I had the same problem, the way to solve and put the root user of mysql instead of putting a created user.

**Comment by ludralph** [comment]
Hi @AndersonVilela 👋 

Can you please use [Google translate](https://translate.google.com/) to translate your question to English. Also, are you referring to a previous discussion or issue? Can you please help with the link.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
